### PR TITLE
Add nether supply ships

### PR DESCRIPTION
**Pull request by ShadowProtocol**
For AravanFox *et al.* builders

# Changes proposed in this pull request:
- Searches for schematics in ``nethership`` folders.
- Only netherships will work in the nether, and supplyships will only work in the overworld
- Netherships require a pool of lava to be placed in
- NOTE: not sure what the story is with the End, so I didn't really account for it.

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/5771/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -694,6 +694,7 @@
 
   "item.supplycampdeployer.invalid": "Requires a flat ground at least 16*17 blocks without grass, flowers, or holes!",
   "item.supplycampdeployer.invalid.water_block_needed": "Placing a Supply Ship requires water under the ship. These blocks must be water: %s.",
+  "item.supplycampdeployer.invalid.lava_block_needed": "Placing a Supply Ship requires lava under the ship. These blocks must be lava: %s.",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "item.supplycampdeployer.invalid.lava_block_needed": "Placing a Supply Ship in the Nether requires lava under the ship. These blocks must be lava: %s.",
```

**Review by Mekle001 — DISMISSED** [review]

**Review by marchermans — CHANGES_REQUESTED** [review]
This looks in general to be okey.
We should however make a note to figure this out in a more general fashion.

**Review by marchermans — APPROVED** [review]
Request changes was wrong....
Wanted to approve.

**Pull request merged**